pallets/staking/src/lib.rs

```diff
@@ -485,8 +485,6 @@ pub struct ValidatorPrefs {
     pub commission: Perbill,
 }
 
-// TODO: Need to be removed before mainnet launch.
-// Keeping this to support the `on_runtime_upgrade`.
```

Comment:
This enum is no longer used outside the storage migration so perhaps we should keep this todo around as a reminder to clean up later.

Reply:
Added a comment